### ISSUE TITLE
CORS policy blocking loading of some CDN JS?

### ISSUE THREAD

**Issue by MattWilcox** [pikapkg/cdn#168]
I'm not 100% sure, but doesn't this meant that the CORS policy on the CDN itself is wrong?

```
Access to script at 'https://cdn.skypack.dev/-/@pixi/text@v6.0.2-uVh8V9loC8gHOm5xAUtX/dist=es2020,mode=imports/optimized/@pixi/text.js' from origin 'https://mattwilcox.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by liuxilu** [comment]
It happened several times since I started to use skypack one month ago.
It's working again after accessing through a proxy. (My local page itself is not proxied).
```
<script type="module">
import { Octokit } from "https://cdn.skypack.dev/@octokit/rest";
</script>
```
This time, it pended for 1.3min and blocked as below .
> Access to script at 'https://cdn.skypack.dev/-/@octokit/request-error@v2.0.5-VXCKzL3tDqJTDyO6Nga3/dist=es2020,mode=imports/optimized/@octokit/request-error.js' from origin 'http://127.0.0.1:5000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Comment by MattWilcox** [comment]
I've been seeing this on and off throughout the week, and I have changed nothing at all.
This makes relying on Skypack completely off the table, as it simply doesn't work some of the time.

**Comment by MattWilcox** [comment]
Yeah, this is untennable. My site has worked half the time I visit it, and half not. I've changed nothing. Whenever not, it's screaming these sort of errors:

```
Loading module from “https://cdn.skypack.dev/-/@pixi/filter-displacement@v6.0.3-Dlu1Oml6RcL5bkxFAdth/dist=es2020,mode=imports/optimized/@pixi/filter-displacement.js” was blocked because of a disallowed MIME type (“text/html”).

Loading module from “https://cdn.skypack.dev/-/@pixi/utils@v6.0.3-s8mWBJru1c3ERxN9tuq7/dist=es2020,mode=imports/optimized/@pixi/utils.js” was blocked because of a disallowed MIME type (“text/html”).

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://cdn.skypack.dev/-/@pixi/filter-displacement@v6.0.3-Dlu1Oml6RcL5bkxFAdth/dist=es2020,mode=imports/optimized/@pixi/filter-displacement.js. (Reason: CORS request did not succeed).

Module source URI is not allowed in this document: “https://cdn.skypack.dev/-/@pixi/filter-displacement@v6.0.3-Dlu1Oml6RcL5bkxFAdth/dist=es2020,mode=imports/optimized/@pixi/filter-displacement.js”.

Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://cdn.skypack.dev/-/@pixi/utils@v6.0.3-s8mWBJru1c3ERxN9tuq7/dist=es2020,mode=imports/optimized/@pixi/utils.js. (Reason: CORS request did not succeed).

Module source URI is not allowed in this document: “https://cdn.skypack.dev/-/@pixi/utils@v6.0.3-s8mWBJru1c3ERxN9tuq7/dist=es2020,mode=imports/optimized/@pixi/utils.js”.
```

**Comment by matthewp** [comment]
Hi @MattWilcox, sorry for not responding here previously. We have been tracking an issue with Cloudflare that has been causing seemingly random outages for some users. I didn't realize before that this issue is probably related to that. You are getting the text/html mime types when a Cloudflare error page is shown.

We've been in contact with their support and are working on fully resolving this. I'll update this issue when we think it's been taken care of. Sorry for the inconvenience.

**Comment by drwpow** [comment]
Update: unfortunately the issue is ongoing, with Cloudflare and Vercel each pointing a finger at the other. If this doesn’t resolve, our only course of action would be to drop one of those providers and migrate the service.

**Comment by MattWilcox** [comment]
Thanks for the update

**Comment by Tonksthebear** [comment]
I just found this thread due to experiencing a round of CORS issues. I’ve been using Skypack for a couple months, and this is the first time I’ve seen the issue. Now I’m worried this might pop up occasionally if I depend on Skypack for my production apps. 

Are there any updates to the situation?

**Comment by Potherca** [comment]
Just chiming in that I am also running into this behavior and will also need to use another provider for NPM packages that need a build step.

I love the idea, but things need to be more stable before I'd feel comfortable even using this in development.
